tasks/cat2.yml

```diff
@@ -3140,3 +3140,33 @@
       - V-51363
       - selinux
 
+- name: "MEDIUM | V-57569 | AUDIT | The noexec option must be added to the /tmp partition."
+  command: egrep '^([^#[:space:]]+[[:space:]]+/tmp[[:space:]]+[^[:space:]]+[[:space:]]+)([^[:space:]]+)([[:space:]]+.*)$' /etc/fstab
+  register: fstab_tmp_audit
+  changed_when: false
```

Comment:
changed_when and failed_when should use `no` instead of `false` but I wouldnt hold up merging over this